Realm/Realm.Unity/README.md

```diff
@@ -0,0 +1,42 @@
+![Realm](https://github.com/realm/realm-dotnet/raw/master/logo.png)
+
+Realm is an embedded mobile database that runs directly on phones, tablets, or desktops.
```

Comment:
@mongomoe tagging you to review the Readme - this is what's going to be shown on npmjs - e.g. here: https://www.npmjs.com/package/moment as well as if the user clicks on it in the unity package manager.